docs/source/recipes.rst

```diff
@@ -334,10 +298,12 @@ If you need to reuse and override existent recipe call extend method:
 
 .. code-block:: python
 
-    dog = Recipe(Dog,
-        breed = 'Pug',
-        owner = foreign_key(person)
+    customer = Recipe(
+        Customer,
+        bio='Some customer bio',
+        age=30,
+        happy=True,
     )
-    extended_dog = dog.extend(
-        breed = 'Super basset',
+    sad_customer = customer.extend(
```

Comment:
I didn't know that :scream: Cool

Reply:
Me neither HAHAHAHAHAHA

The `extend` automatically became one of my favorites hidden features =]